### PR TITLE
Fix bug where the file handles were no longer put on the command line

### PR DESCRIPTION
**Pull request by rseanhall**
While debugging this, I found another issue - apparently the new way of launching the process with the file handles causes the Executable path to be in wzCommandLine so there's an extra argument now.  We'll have to fix that, too.

### CODE REVIEW

File: src/burn/engine/engine.cpp [hunk 1]
```diff
@@ -435,21 +439,21 @@ static HRESULT RunUntrusted(
 
     // The clean room switch must always end up at the front of the command line so
     // the EngineInCleanRoom function will operate correctly.
-    hr = StrAllocFormattedSecure(&sczParameters, L"-%ls=\"%ls\" %ls", BURN_COMMANDLINE_SWITCH_CLEAN_ROOM, sczCurrentProcessPath, wzCommandLine);
-    ExitOnFailure(hr, "Failed to allocate parameters for unelevated process.");
+    hr = StrAllocFormattedSecure(&sczFullParameters, L"-%ls=\"%ls\" %ls", BURN_COMMANDLINE_SWITCH_CLEAN_ROOM, sczCurrentProcessPath, sczParameters);
+    ExitOnFailure(hr, "Failed to append %ls", BURN_COMMANDLINE_SWITCH_CLEAN_ROOM);
```
Comment (robmen) [inline src/burn/engine/engine.cpp]: `prepend` not `append`, right?

**Comment by robmen** [comment]
Do you want me to take this (with the logging fix) or should we wait for the extra argument fix as well?

File: src/burn/engine/engine.cpp (hunk 1 above)
Comment (rseanhall) [inline src/burn/engine/engine.cpp]: Yeah, all three of these should be prepend.

**Comment by rseanhall** [comment]
I'll file a bug for both of these and resubmit.

**Pull request closed without merging**